### DISCUSSION TITLE
Change animation speed of slide_from_right android

### DISCUSSION THREAD

**Discussion by MetinArslanturk** [software-mansion/react-native-screens#893] [Q&A] [answered]
Hello guys, really good library thanks for your effort. I have a question, i want to change slide_from_right animation's speed in android.  I see it set "config_mediumAnimTime" in https://github.com/software-mansion/react-native-screens/blob/master/android/src/main/res/anim/rns_slide_in_from_right.xml
How can i override this? Is it possible? Or i need to clone project? Thanks.

**Comment by kacperkapusciak** [comment, marked as answer]
Hi @MetinArslanturk,
Currently, we don't have a way to customize animation speed but you should be able to easily overwrite it with [patch-package](https://www.npmjs.com/package/patch-package).
[This guide](https://www.roshan.info.np/posts/patch-package-nodemodules) should be helpful.
Cheers!
